### ISSUE TITLE
Question with podcast:locked and independent podcasters

### ISSUE THREAD

**Issue by cio-blubrry** [Podcastindex-org/podcast-namespace#168]
Regarding [podcast:locked tag](https://github.com/Podcastindex-org/podcast-namespace/blob/main/docs/1.0.md#locked) and working with independent podcasters, podcasters who are not using a managed platform. In our case, podcasters who host their own podcast on their own website using WordPress with PowerPress. In these cases there is no way to provide a email verification system as one would be verifying themselves.

Currently the wording states "The purpose is to tell other podcast platforms whether they are allowed to import this feed. A value of yes means that any attempt to import this feed into a new platform should be rejected." This is fine for an independently managed (not tied to a service) podcast. If yes is set, we will not allow the podcaster to import.

The last part "When importing a feed, if the hosting provider has already verified the owner="" email address on their own system, and the email matches what is listed in this tag, it is safe to import the feed.", this assumes that the podcaster is on a platform with a mechanism to verify the email address. In the situation where they are self hosted, they would be verifying themselves. Further, in the 'owner' parameter it states "The owner attribute is an email address that **can be** used to verify ownership of this feed during move and import operations.". I assume since the "can be" is in this line, for the situation we are handling, we can ignore the owner email field and simply use the tag value of "no" to indicate that the feed can be imported.

In other words: <podcast:locked>yes</podcast:locked> PowerPress will not allow the podcaster to import the feed. If <podcast:locked>no</podcast:locked> or not set, PowerPress will allow the podcaster to import the feed.

Some clarity and confirmation our approach is acceptable is appreciated.

**Comment by tomrossi7** [comment]
> In other words: <podcast:locked>yes</podcast:locked> PowerPress will not allow the podcaster to import the feed. If <podcast:locked>no</podcast:locked> or not set, PowerPress will allow the podcaster to import the feed.

That seems like a fine approach! @daveajones do you see any issues with it?

**Comment by daveajones** [comment]
Yes that is totally fine.  I will also add wording to this effect to clarify. Thanks for the thorough explanation of this scenario.

**Comment by Marzal** [comment]
This was fixed with https://github.com/Podcastindex-org/podcast-namespace/pull/171

New path after refactor:
https://github.com/Podcastindex-org/podcast-namespace/blob/main/docs/tags/locked.md
